src/bodywork/k8s/workflow_jobs.py

```diff
@@ -72,8 +72,15 @@ def configure_workflow_job(
         image=image,
         image_pull_policy="Always",
         env=vcs_env_vars,
-        command=["bodywork", "workflow"],
-        args=[project_repo_url, project_repo_branch],
+        command=[
```

Comment:
The command is `bodywork deployment create`, but the rest are args, no?

There is a reason why you should split things in this way, but I'm failing to remember my Docker 101, etc.

Reply:
Hmmm, I thought we couldn't split it this way because they need the flags but then it might work just adding the flags to the args. 

I'd need to try it out, if it does work then that's probably better, I don't know why I didn't try that out first. I shouldn't code late evening.

Reply:
Hit the k8s book in the morning, or perhaps better might be the official [Dockerfile best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/).

Reply:
And yeah, you shouldn't be coding late at night - that's the sort of stupid sh*t that I do...